src/tests/posix/utime_test.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file
+ * @brief test for utime
+ *
+ * @author  Filipp Chubukov
+ * @date    21.04.2019
+ */
+
+#include <fcntl.h>
+#include <utime.h>
+#include <sys/stat.h>
+#include <embox/test.h>
+#include <errno.h>
+
+#define TEST_EXIST_FILE "test"
+#define TEST_NOT_EXIST_FILE "not_exist_test"
+EMBOX_TEST_SUITE("posix/utime_test");
+
+TEST_CASE("check changed time") {
+	struct utimbuf prev_time;
+	struct utimbuf new_time;
+
+	struct stat filestat;
+
+	new_time.actime = 999;
+	new_time.modtime = 999;
+
+	creat(TEST_EXIST_FILE, 0666);
```

Comment:
Check for error code? Same for other posix calls

Reply:
added